buildSrc/src/main/kotlin/io/tnboot/gradle/build/mavenPublish.kt

```diff
@@ -6,8 +6,6 @@ import org.gradle.api.publish.maven.MavenPublication
 import org.gradle.api.publish.plugins.PublishingPlugin
 import java.net.URI
 
-// todo(James Bradlee): Replace this with io.tnboot's GitHub plugin
-//                      once it's been implemented.
```

Comment:
Because this is evaluated before any other project, it is impossible to re-use the contents inside of the gradle-plugin project without doing nasty things. We'll keep this as-is and change it later if it becomes a necessary need.